Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -559,7 +569,7 @@ def tc_indicator_get_tags(tc, owners, indicator_value, indicator_type):
     ro.set_http_method('GET')
     ro.set_owner(owners)
     if apiBranch is not None:
-        ro.set_request_uri("/v2/indicators/{}/{}/tags".format(apiBranch, indicator_value))
+        ro.set_request_uri("/v2/indicators/{}/{}/tags".format(apiBranch, quote(indicator_value).replace("/", "%2F")))
```

Comment:
URL Encode indicator value to extract indicator tags properly (Only effects hosts and URLs)

Reply:
Might have to rethink the methodology on this, the URL and Host Indicators follow RFC 3986, but not all indicator types do. For example http://domain.com with the current implement will error as there is no data, but with my version will give the correct tags. However, with the indicator type EmailAddress the @ is not encoded.

Reply:
As for the .replace(), I saw that used earlier in the code base and was trying to follow the same structure. There is no difference between using safe='' and .replace("/", "%2F")

Reply:
Example:

Current Master Branch Output:
![image](https://user-images.githubusercontent.com/78560684/114061331-53804980-9864-11eb-82b1-34cb3c851680.png)

My Version Output:
![image](https://user-images.githubusercontent.com/78560684/114061414-6bf06400-9864-11eb-89a6-ef1e1926d7c8.png)

Reply:
A safer solution could potentially be:

Line +26
`
ENCODED_API_BRANCHES = ["urls", "hosts"]
`
*
*
*

Line 530
`
        if apiBranch in ENCODED_API_BRANCHES:
            indicator_value = quote(indicator_value, safe='')
`